ui/app/common/patient-search/controllers/patientsListController.js

```diff
@@ -40,7 +40,8 @@ angular.module('bahmni.common.patientSearch')
             if (searchType.handler) {
                 var params = { q: searchType.handler, v: "full",
                     location_uuid: $bahmniCookieStore.get(Bahmni.Common.Constants.locationCookieName).uuid,
-                    provider_uuid: $rootScope.currentProvider.uuid };
+                    provider_uuid: $rootScope.currentProvider.uuid,
```

Comment:
@tmvumbi2 The tests are failing, please fix.

Reply:
TypeError: $rootScope.currentUser is undefined in /home/travis/build/jembi/cameroon-openmrs-module-bahmniapps/ui/app/common/patient-search/controllers/patientsListController.js (line 9)